server/test/controllers/ControllerTest.scala

```diff
@@ -1,30 +1,34 @@
 package controllers
 
-import java.util.{UUID, Date}
+
+import java.util.{Date, UUID}
 
 import controllers.ControllerTest._
 import dal.Dao
 import models.TaskType._
-import models.{NewTask, Task, User}
+import models.{NewTask, Task}
 import monifu.concurrent.Implicits.globalScheduler
 import monifu.concurrent.Scheduler
 import org.scalamock.scalatest.MockFactory
-import org.scalatestplus.play.PlaySpec
+import org.scalatestplus.play.{OneAppPerSuite, PlaySpec}
 import play.api.test.Helpers._
 import play.api.test._
-import service.{DynamicSuiteExecutor, ScalaTestRunner}
+import service._
 
 import scala.concurrent.Future
 
-class ControllerTest extends PlaySpec with MockFactory {
+class ControllerTest extends PlaySpec with MockFactory with OneAppPerSuite {
```

Comment:
The idea is:
- this suite runs everything w/o App
- so please move everything that need OneAppPerSuite to separate suite or add it to `ApplicationTest extends with OneAppPerSuite`